### PR TITLE
Fixed #27552 -- Added docs/tests for GEOSGeometry.normalize().

### PR DESCRIPTION
**Pull request by sir-sigurd**
https://code.djangoproject.com/ticket/27552

### CODE REVIEW

File: docs/ref/contrib/gis/geos.txt [hunk 1]
```diff
@@ -652,6 +652,18 @@ Other Properties & Methods
             positive integer even if it was called with a
             :class:`~django.contrib.gis.gdal.CoordTransform` object.
 
+.. method:: GEOSGeometry.normalize()
+
+    Converts this geometry to canonical form::
+
+        >>> g = MultiPoint(Point(0, 0), Point(2, 2), Point(1, 1))
+        >>> print(g)
```
Comment (timgraham) [inline docs/ref/contrib/gis/geos.txt]: Is `print(g)` better/different than `g` and relying on `repr()`?

File: docs/ref/contrib/gis/geos.txt [hunk 2]
```diff
@@ -652,6 +652,18 @@ Other Properties & Methods
...
+        MULTIPOINT (0 0, 2 2, 1 1)
+        >>> g.normalize()
+        >>> print(g)
+        MULTIPOINT (2 2, 1 1, 0 0)
+
```
Comment (timgraham) [inline docs/ref/contrib/gis/geos.txt]: 1 blank line between sections

File: docs/ref/contrib/gis/geos.txt (hunk 1 above)
Comment (sir-sigurd) [inline docs/ref/contrib/gis/geos.txt]: Yes, `repr(g)` returns `<MultiPoint object at 0x7fb1682472e0>`.

File: docs/ref/contrib/gis/geos.txt (hunk 2 above)
Comment (sir-sigurd) [inline docs/ref/contrib/gis/geos.txt]: Fixed.

**Pull request merged**